plugins/modules/ec2_spot_instance.py

```diff
@@ -0,0 +1,516 @@
+#!/usr/bin/python
+# This file is part of Ansible
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: ec2_spot_instance
+version_added: 2.0.0
+short_description: request, stop, reboot or cancel spot instance
+description:
+    - Creates, stops, reboots or cancels spot instances.
+    - >
+      Note: This module uses the boto3 Python module to interact with the EC2 API.
+      M(amazon.aws.ec2) will still support the older boto Python module to interact with spot instances.
```

Comment:
We can lose this note.  The `boto` is no longer supported upstream and by adding the document fragment we automatically add a note about which Python modules we require.